radish34/contracts/__tests__/mockVerifierData.js

```diff
@@ -9,62 +9,135 @@
-    "vks_hex": ["0x265f1cf0fdc90efca70e3e568612d681313857b7fab471feb15bd93facd1b2c8",
-        "0x0b75345b01ebbd6479ddfeccd917fecd87be95bd636f701fbe866979ea43d236",
-        "0x26a17095359634b4e4d367d9dc7ad443ae694b5981c6407072d38b369a7142b8",
-        "0x244b5b1f967dc837c7a8fdd8f20d7b8cbaedf31855fbb1fbff451859a265605e",
-        "0x2a43c661bcdd84dc6b73298b8ff7b9a8496e9963d7d9bc35dba959e14064c7f0",
-        "0x27482c89d1b631dd855e808909c42bf73e24b63ebe7bfa5b86a7bdaca54da597",
-        "0x1d2fc947df8859b566f7a92e68aa32e6c1aed8d5033130a726b92017dc129c9f",
-        "0x17189f435d8b65290d526a944c33c8728af7d92bcaa22cdc043593bb6f6bb627",
-        "0x1db2ab3fc775786823e349e476f8806e2b82f2eee4d8c832ceadba0eba2564ae",
-        "0x211232aca4f0ecbfd29247a10b861043acc236d9eb2b254f430e11612ee6900f",
-        "0x13571c4b7143bce87ffd53a27a44ff825be0f6bde8492434181b64c2e065316a",
-        "0x174e184b69450f4e262778160ee55b27788f137f3095d9eabb230f619b89c0bf",
-        "0x0fc187dba209eb0258f9402a0f909efbef59e4633353c6d6ac88dd06fe7e222c",
-        "0x17ba199beabd54568f7c8f08a9e71bca9b58c2868fbd5bea2d826efca9edccd9",
-        "0x07d501e4e6a4a758af65654948eeef89be977e49668fe1066cd7c88f6693902c",
-        "0x29f8a7b0559ca992b55c4f5d11ddd08770027c9064bb9b939a4a53db31d99fa9",
-        "0x261c06397956101ba684a15bfe073593e414d400e2d16f0ffd0c56474b201957",
-        "0x1870e9974943bf37950887e1132a65f11b53089afdf18f1de81de3db2a606042",
-        "0x0337c832500842dcc4eac0fa5cec035915c8fc316da62ec5580de1f75f9d58b0",
-        "0x0cca0d84a3671ec509c0b381b95be13c41d67a0e336743f3eb856184022cbdfc"],
-    "proofs_hex": ["0x2123dfba77aa2b90e480d484667f433f44b83f196ae71b60589d5c7638d23ff5",
-        "0x17618efc12151144cdb4f780b4daf9c7763eed307ebf0825ef819f174587000b",
-        "0x085e05d8276145445345fc2d50caa8e93657c6582175535d61b9f56f7bde997b",
-        "0x1217c4734ddc69ef73168b026cd811bfea87d8b23dd765a2860eb6e47250d63a",
-        "0x0191a2f43eff4e051d619ae69db206e1e671af45ad02c7c6a141a869d2d2c5a9",
-        "0x0186373c53ca801d67ebb7992217330a9b81ba3c5f03165c992fc80ca16a2f80",
-        "0x2eb76d6483ebd9e8a02a48aa1e343bef331428cccfdd3f1829c943dc50c1bc19",
-        "0x020ca55394bdb87edfbcb54e6961cac54f83167d7e1574e60b00d7e0ffedeac4" ],
-    "proofs_int": ["14989707767102615313684605173057903641347878660550199499377470094238716936181",
-        "10575566526490722493368634185430526475607354877647401472809285966069569617931",
-        "3784626748982707459523154788722591507947491336303157290452437655043216152955",
-        "8183624607866254943801646970430624514997174711794003680811213610669243946554",
-        "709630340745687344505736606096953814072381190831471097905206887612184446377",
-        "689451577733667459872895366744634263233388447995637921789623037416555818880",
-        "21130479047924098851097522361915995958233452872465264954591872849418039442457",
-        "926968903430932935510549259338269173796291853932043469374211440611536267972"],
+  "msa": {
```

Comment:
Why did these change?

Reply:
during testing, we needed to generate new sets. ideally, we could something like `axios` to mock this data

Reply:
Previously this file only contained mock data for the `createMSA` transactions. I added some nesting so that the data is separated for `msa` and `po`. So the changes to this file are to nest all the existing data in an `msa` object and add similar data in a new `po` object

Reply:
okay thanks for the explanation 👍